src/annotator/integrations/vitalsource.ts

```diff
@@ -22,11 +22,41 @@ import type { InjectConfig } from '../hypothesis-injector';
 // smaller and it feels unreadable or too-zoomed-out
 const MIN_CONTENT_WIDTH = 480;
 
+/**
+ * Book metadata exposed by the VitalSource viewer.
+ */
+type BookInfo = {
+  /**
+   * Indicates the book type. "epub" means the book was created from an EPUB
+   * and the content is XHTML. "pbk" means the book was created from a PDF and
+   * has a fixed layout.
+   */
+  format: 'epub' | 'pbk';
+
+  /**
+   * VitalSource book ID ("vbid"). This identifier is _usually_ the book's
+   * ISBN, hence the field name. However this value is not always a valid ISBN.
+   */
+  isbn: string;
+  title: string;
+};
+
+/**
+ * `<mosaic-book>` custom element in the VitalSource container frame.
```

Comment:
Could you amend this comment such that it is very clear that `<mosaic-book>` is part of VitalSource content documents/their API?